### PR TITLE
installing 3rd party software

### LINKED ISSUE #450: Installing the 3rd Party Softwares sent from the webapp into the ISO
- Issue type: Feature request 

### Short description

In the current build, recipe array is sent to the build server from the meilix-generator now as an environment variable of the build server, Next steps would be installing the sent packages(chosen by the user) into the Meilix ISO.
The user has the options to choose from a set of packages and then it needs to be installed into the ISO.

### Usecase

This feature will help to create custom distributions which will have the 3rd party softwares that needs to be pre-installed that to choosen by the user. The various options can be seen by seeing the meilix-generator.

### PR DESCRIPTION
**Pull request by rpotter12**
### Short description
these changes will install the 3rd party software on the basis of selected checkboxes in the meilix-generator.
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Fixes #450

### CODE REVIEW

File: scripts/chroot_reconst.sh [hunk 1]
```diff
@@ -58,37 +58,52 @@ rm  /usr/share/xsessions/plasma.desktop
...
+	then
+		wget -q https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb &&
+		dpkg -i google-chrome-stable_current_amd64.deb &&
+		rm -f google-chrome-stable_current_amd64.deb
+	else
+		wget -q https://dl.google.com/linux/direct/google-chrome-stable_current_i386.deb &&
+		dpkg -i google-chrome-stable_current_i386.deb &&
+		rm -f google-chrome-stable_current_i386.deb
+	fi
+else if ["vlc" in "$INSTALL"]
```
Comment (abishekvashok) [inline scripts/chroot_reconst.sh]: What if both vlc and git are in install only vlc will be installed. Maybe, try replacing else if s with if s

Comment (rpotter12) [reply scripts/chroot_reconst.sh]: yep, there should not be any else if part. Thanks :)

**Comment by rpotter12** [comment]
@abishekvashok please review it again

**Comment by rpotter12** [comment]
Sorry I missed that

**Comment by rpotter12** [comment]
@tabesin @abishekvashok @meets2tarun @xeon-zolt please review it again

**Pull request merged**